### PR TITLE
Accessibility Issue creation

### PR DESCRIPTION
**Pull request by Truealex4**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,6 +29,12 @@ Eligible contributions according to the [Hacktoberfest guidelines](https://hackt
 
 Contributions:
 
+- 2023-10-30 - [Alex Lovell](https://github.com/Truealex4) - Accessibility testing of v10 CMS and raising issue - Issue #15067 https://github.com/umbraco/Umbraco-CMS/issues/15067 - Contrast issue upon successful login landing page
+- 2023-10-30 - [Alex Lovell](https://github.com/Truealex4) - Accessibility testing of v10 CMS and raising issue - Issue #15068 https://github.com/umbraco/Umbraco-CMS/issues/15068 - Media library is missing alt text attribute
+- 2023-10-30 - [Alex Lovell](https://github.com/Truealex4) - Accessibility testing of v10 CMS and raising issue - Issue #15069 https://github.com/umbraco/Umbraco-CMS/issues/15069 - Umbraco logo is missing alt attribute
+- 2023-10-30 - [Alex Lovell](https://github.com/Truealex4) - Accessibility testing of v10 CMS and raising issue - Issue #15070 https://github.com/umbraco/Umbraco-CMS/issues/15070 - contrast issue on 'Login' CTA
+- 2023-10-30 - [Alex Lovell](https://github.com/Truealex4) - Accessibility testing of v10 CMS and raising issue - Issue #15071 https://github.com/umbraco/Umbraco-CMS/issues/15071 - 'Select All' button has no accessible name
```
Comment (busrasengul) [inline README.md:36]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -29,6 +29,12 @@ Eligible contributions according to the [Hacktoberfest guidelines](https://hackt
 
 Contributions:
 
+- 2023-10-30 - [Alex Lovell](https://github.com/Truealex4) - Accessibility testing of v10 CMS and raising issue - Issue #15067 https://github.com/umbraco/Umbraco-CMS/issues/15067 - Contrast issue upon successful login landing page
+- 2023-10-30 - [Alex Lovell](https://github.com/Truealex4) - Accessibility testing of v10 CMS and raising issue - Issue #15068 https://github.com/umbraco/Umbraco-CMS/issues/15068 - Media library is missing alt text attribute
+- 2023-10-30 - [Alex Lovell](https://github.com/Truealex4) - Accessibility testing of v10 CMS and raising issue - Issue #15069 https://github.com/umbraco/Umbraco-CMS/issues/15069 - Umbraco logo is missing alt attribute
+- 2023-10-30 - [Alex Lovell](https://github.com/Truealex4) - Accessibility testing of v10 CMS and raising issue - Issue #15070 https://github.com/umbraco/Umbraco-CMS/issues/15070 - contrast issue on 'Login' CTA
```
Comment (busrasengul) [inline README.md:35]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -29,6 +29,12 @@ Eligible contributions according to the [Hacktoberfest guidelines](https://hackt
 
 Contributions:
 
+- 2023-10-30 - [Alex Lovell](https://github.com/Truealex4) - Accessibility testing of v10 CMS and raising issue - Issue #15067 https://github.com/umbraco/Umbraco-CMS/issues/15067 - Contrast issue upon successful login landing page
+- 2023-10-30 - [Alex Lovell](https://github.com/Truealex4) - Accessibility testing of v10 CMS and raising issue - Issue #15068 https://github.com/umbraco/Umbraco-CMS/issues/15068 - Media library is missing alt text attribute
+- 2023-10-30 - [Alex Lovell](https://github.com/Truealex4) - Accessibility testing of v10 CMS and raising issue - Issue #15069 https://github.com/umbraco/Umbraco-CMS/issues/15069 - Umbraco logo is missing alt attribute
```
Comment (busrasengul) [inline README.md:34]:
```suggestion
```

File: README.md [hunk 4]
```diff
@@ -29,6 +29,12 @@ Eligible contributions according to the [Hacktoberfest guidelines](https://hackt
 
 Contributions:
 
+- 2023-10-30 - [Alex Lovell](https://github.com/Truealex4) - Accessibility testing of v10 CMS and raising issue - Issue #15067 https://github.com/umbraco/Umbraco-CMS/issues/15067 - Contrast issue upon successful login landing page
+- 2023-10-30 - [Alex Lovell](https://github.com/Truealex4) - Accessibility testing of v10 CMS and raising issue - Issue #15068 https://github.com/umbraco/Umbraco-CMS/issues/15068 - Media library is missing alt text attribute
```
Comment (busrasengul) [inline README.md:33]:
```suggestion
```

File: README.md [hunk 5]
```diff
@@ -29,6 +29,12 @@ Eligible contributions according to the [Hacktoberfest guidelines](https://hackt
 
 Contributions:
 
+- 2023-10-30 - [Alex Lovell](https://github.com/Truealex4) - Accessibility testing of v10 CMS and raising issue - Issue #15067 https://github.com/umbraco/Umbraco-CMS/issues/15067 - Contrast issue upon successful login landing page
```
Comment (busrasengul) [inline README.md:32]:
```suggestion
- 2023-10-30 - [Alex Lovell](https://github.com/Truealex4) - Accessibility testing of v10 CMS and raising issues - [Issues](https://github.com/umbraco/Umbraco-CMS/issues?q=is%3Aissue+author%3ATruealex4+created%3A2023-10-01..2023-10-31+accessibility)
```

**Review by busrasengul — CHANGES_REQUESTED** [review]
Let's merge all the issues in one place and link them to the relevant link in the issue tracker :)
This way it's easier to review for all the issues!
Thanks a lot @Truealex4

**Comment by busrasengul** [comment]
Perfect now! Let's get this merge in 🎉

**Comment by Alex-Lovell** [comment]
Hi Seb,

Yeah, I got some feedback that people who didn’t know who the real Alex Lovell was!

Have a great weekend.

Alex

**Pull request merged**